src/screens/PickList/index.tsx

```diff
@@ -360,8 +367,12 @@ const PickOrderItem = () => {
     onBinLocationBarCodeSearchQuerySubmitted();
   };
 
-  const quantityPickedChange = (query: string) => {
-    setState({
+  const quantityPickedChange = (query: string, index: number) => {
+    let picklistItemData = pickListItemData;
+    picklistItemData[index].quantityToPick = parseInt(query); 
+    setPickListItemData([ ...picklistItemData ]);
+    
+setState({
```

Comment:
The indentation looks a bit off.

Reply:
Sorry @awalkowiak i didn't understand your comment, can you please explain me in details